examples/platforms/da15000/cxx-symbols.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
It will be just file rename, since we don't have startup-gcc.c in our solution. But OK, it can be done.